docs/framework/network-programming/tls.md

```diff
@@ -53,10 +53,15 @@ For more information see [TLS protocol version support in Schannel](/windows/win
 
 ## Recommendations
 
-- For TLS 1.3, target .NET Framework 4.8 or later.
-- Do not specify the TLS version explicitly. Configure your code to let the OS decide on the TLS version.
+- For TLS 1.3, target .NET Framework 4.8 or later. ([here](#audit-your-code-and-make-code-changes) is how to check what is your `target framework`.)
+- Do not specify the TLS version explicitly. (i.e. don't use the method overloads of SslStream that take an explicit SslProtocols parameter.)
+  - That way your code will let the OS decide on the TLS version.
+  - If you must set HttpWebRequest.SslProtocols, then set it to `SslProtocols.SystemDefault`. That will also use OS default.
```

Comment:
There should probably be clickable link to HttpWebRequest.SslProtocols via xref

Reply:
Good catch, thanks!